### PR TITLE
Provide some initial test coverage

### PR DESCRIPTION
**Pull request by lauriii**

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-import ReactDOM from 'react-dom';
+import { shallow } from 'enzyme';
+
 import App from './App';
 
 it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+  const root = shallow(<App />);
```
Comment (dawehner) [inline src/App.test.js]: I'm curious, what happens with the IO that should be fired here?

File: src/components/05_pages/Roles/Roles.test.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    // Ensure that task is cancelled on unmount.
+    root.unmount();
+    expect(cancelTask).toHaveBeenCalled();
+  });
+
+  it('loading animation is shown before roles have been loaded', () => {
+    const root = shallow(
+      <Roles cancelTask={() => {}} requestRoles={() => {}} roles={null} />,
+    );
+    expect(root.find(LoadingBar).length).toEqual(1);
```
Comment (dawehner) [inline src/components/05_pages/Roles/Roles.test.js]: That is super nice!

File: src/App.test.js (hunk 1 above)
Comment (lauriii) [reply src/App.test.js]: This actually only renders the main level so we don't have to worry about that as part of this. We should solve problems related to that in a follow-up issue

Comment (dawehner) [reply src/App.test.js]: Ah gotcha, nevermind :)

File: src/routes.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import Permissions from './components/05_pages/Permissions/Permissions';
-import Roles from './components/05_pages/Roles/Roles';
+import Roles from './components/05_pages/Roles';
```
Comment (dawehner) [inline src/routes.js]: Works for me!

**Comment by mattgrill** [comment]
Can we add a task to Circle CI to run these?

File: src/components/06_wrappers/ErrorBoundary/ErrorBoyndary.test.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```
Comment (larowlan) [inline src/components/06_wrappers/ErrorBoundary/ErrorBoyndary.test.js]: filename has a typo here

Comment (dawehner) [reply src/components/06_wrappers/ErrorBoundary/ErrorBoyndary.test.js]: I fixed that in https://github.com/jsdrupal/drupal-admin-ui/pull/73

**Pull request merged**